napari/layers/points/_tests/test_points.py

```diff
@@ -443,13 +443,19 @@ def test_remove_selected_updates_value():
     data = 20 * np.random.random(shape)
     layer = Points(data)
 
+    layer.events.data = Mock()
     # set the value
     layer._value = 3
     layer._value_stored = 3
 
-    layer.selected_data = {0, 5, 6, 7}
+    selection = {0, 5, 6, 7}
+    layer.selected_data = selection
     layer.remove_selected()
-
+    assert layer.events.data.call_args[1] == {
+        "value": layer.data,
+        "action": "remove",
+        "data_indices": list(selection),
```

Comment:
I do not like this part of the comparison. Conversion to list enforces order. I prefer to evet use frozenset for such purposes. As semantically this is not mutable unordered collection of points

Reply:
resolved with [d636cc8](https://github.com/napari/napari/pull/5967/commits/d636cc84bf628245473ccdea9739c1be13b741b0)